tests/functional/ext/docparams/raise/missing_raises_doc_exc_inheritance.rc

```diff
@@ -0,0 +1,2 @@
+[MASTER]
+load-plugins = pylint.extensions.docparams
```

Comment:
What happens if we add:
```ini
[BASIC]
accept-no-raise-doc=no
```

Reply:
This follows all the other tests. Why making another think here?

Reply:
Some of the other tests also use this setting. I think since we're specifically looking at `raise` documentation I think it makes sense to require it.

Reply:
Now I get
```
self = <pylint.testutils.lint_module_test.LintModuleTest object at 0x7f8c00d26e10>

    def runTest(self) -> None:
>       self._runTest()
E       AssertionError: Wrong results for file "missing_raises_doc_exc_inheritance":
E       
E       Expected in testdata:
E         12: missing-raises-doc

pylint/testutils/lint_module_test.py:88: AssertionError
```

Reply:
Guess we found a bug then. I think with requiring `raises` doc we should still expect one `missing-raises-doc` message.

Reply:
I don't understand the config. Can you please explain what it does and what I need to change in the code?

Reply:
With `missing-raises-doc=no` docstrings are required to document all exceptions that are found within the function or class definition. Since we are already (supposedly) documenting the exception this shouldn't create a new test failure so this is probably a bug in the current implementation of this feature.

Reply:
Sorry, my bad. I have read replace instead of add. Adding these two lines changes nothing. Tests are fine

Reply:
Yes please! Could you then also update the test file names to include `required`? You can look at the other similar files in the directory.